web/src/components/dataConfigForm/DatabaseConfigFormAutomatic.tsx

```diff
@@ -45,30 +45,22 @@ export const DatabaseConfigForm = ({
   return (
     <Stack spacing={4}>
       <ConfigInput
-        label="Host & Port"
+        label="Workspace Host"
         placeholder="http://127.0.0.1:8808"
```

Comment:
nit: The placeholder shouldn't contain the port if we are no longer asking for it.

```suggestion
        placeholder="http://127.0.0.1"
```